app/src/androidTest/java/woowacourse/omok/DBTest.kt

```diff
@@ -0,0 +1,46 @@
+package woowacourse.omok
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import omok.model.entity.Point
+import omok.model.entity.Stone
+import omok.model.entity.StoneColor
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import woowacourse.omok.db.OmokDbHelper
+import woowacourse.omok.db.StoneDao
+
+class DBTest {
+    private lateinit var stoneDao: StoneDao
+
+    @Before
+    fun setUp() {
+        val context = ApplicationProvider.getApplicationContext<Context>()
+        stoneDao = StoneDao(context, dbHelper = OmokDbHelper(context, "TEST"))
+    }
+
+    @After
+    fun tearDown() {
+        stoneDao.drop()
+    }
+
+    @Test
+    fun storeOneStone() {
+        val point = Point(1, 1)
+        val expected = Stone(point, StoneColor.BLACK)
+        stoneDao.save(expected)
+        val actual = stoneDao.findAt(point)
+        assertThat(actual).isEqualTo(expected)
+    }
```

Comment:
```suggestion
    @Test
    fun storeOneStone() {
        val point = Point(1, 1)
        val expected = Stone(point, StoneColor.BLACK)
        stoneDao.save(Stone(point, StoneColor.WHITE))
        stoneDao.save(expected)
        val actual = stoneDao.findAt(point)
        assertThat(actual).isEqualTo(expected)
    }
```

현재 중복된 Point 에 서로 다른 stoneColor 를 세팅할 수 있어서,
이런식으로 테스트를 하면 테스트가 터집니다.

직관적으로 생각하면, 같은 좌표가 WHITE 로 설정되어 있더라도,
나중에 BLACK 으로 save 를 하면 INSERT 가 아니라 update 처리가 되어 색깔이 바뀌어야 합니다.
하지만 테스트 실패 내용을 보면 그대로 WHITE 인 색깔을 반환하고 있는 것을 알 수 있습니다.

이 문제를 해결해주세요.